src/views/SubjectAllQuestions/Question/Question.tsx

```diff
@@ -108,6 +136,19 @@ export const Question = ({
                 <AddedByZdajSeIcon className={classes.icon} />
               </Tooltip>
             )}
+            {learntButton && (
+              <Paper
+                onClick={() => learntButton.onClick(!learntButton.checked)}
+                variant="outlined"
+                className={classes.learntWrapper}
+              >
+                <SchoolIcon fontSize="small" />
+                <Checkbox
+                  checked={learntButton.checked}
+                  className={classes.learntCheckbox}
+                />
+              </Paper>
+            )}
```

Comment:
I really enjoy this UI 🚀 I have a few notes though:
1. Instead of using `Paper` let's change it to `Button`
2. Let's make the `Button` a `label` (`component="label"`)
3. Let's move the interaction from the wrapper's on click, to checkbox's `onChanged`
4. Let's add `aria-label` to the checkbox
5. Let's disable completely the ripple effect on the checkbox (even the hover effect)
6. Let's add a tooltip to the whole thing. Maybe: "Oznacz jako nauczone"